src/pages/workspace/AccessOrNotFoundWrapper.tsx

```diff
@@ -122,7 +122,7 @@ function AccessOrNotFoundWrapper({accessVariants = [], fullPageNotFoundViewProps
 
     const shouldShowFullScreenLoadingIndicator = !isMoneyRequest && isLoadingReportData !== false && (!Object.entries(policy ?? {}).length || !policy?.id);
 
-    const isFeatureEnabled = featureName ? PolicyUtils.isPolicyFeatureEnabled(policy, featureName) : true;
+    const isFeatureEnabled = true;
```

Comment:
Was this temporary for testing or intentionally removed?

Reply:
yes, temporary. Sorry, I've overlooked that

Reply:
All good